dv/uvm/sim.py

```diff
@@ -222,7 +189,17 @@ def compare(test_list, iss, output_dir, verbose):
         else:
           print("Unsupported ISS" % iss)
           sys.exit(1)
-        compare_trace_csv(rtl_csv, iss_csv, "ibex", iss, report)
+        # construct trace script call from list of strings and call run_cmd()
+        compare_cmd = ("python3 %s/instr_trace_compare.py " % gen_dir) + \
```

Comment:
What's the reason for this change? I'd prefer just do a functional call rather than running a bash command.

Reply:
I changed this to match the rest of the various simulation scripts, and just using a functional call makes argument parsing more wordy, as well as the fact that the argparser  in instr_trace_compare.py will not be needed with that approach

Reply:
I think each python script should be able to run independently(argparser is needed in this case), and it can also be included in other python program.  I think we have already avoided running another python script with a bash command. The run_cmd is used for non-python programs.